apps/site/lib/site_web/templates/event/_calendar.html.eex

```diff
@@ -24,8 +24,8 @@
             <% todays_events = Map.get(events_by_day, day.day) %>
             <%= if todays_events != nil do %>
               <%= for event <- todays_events do %>
-                <% event_ended = event_ended(%{start: event.date, stop: event.date_end}) %>
-                <button class="m-event-calendar__event" data-a11y-dialog-show="<%= event.id %>">
+                <% event_ended = event_status(%{start: event.date, stop: event.date_end}) === :ended %>
```

Comment:
`===` is used more for comparing numbers, so `==` would also work here (no need to change if you don't want to, just something to keep in mind).

Reply:
I can be sloppy with my equal signs!  😅  To learn more about the difference, did a lil search and found (this helpful reference)[https://inquisitivedeveloper.com/lwm-elixir-7/#:~:text=Like%20Javascript%2C%20Elixir%20has%20two,and%20triple%20equals%20(%20%3D%3D%3D%20).&text=The%20triple%20equals%20operator%20(%20%3D%3D%3D,are%20the%20exact%20same%20value.].  It seems they differ in strictness level.  So since I'm working with atoms and not floats, I think `===` might be the better fit.  But that was _definitely not_ an educated choice when I made it! :joy: